io.github.lenucksi.SieveEditor.yml

```diff
@@ -0,0 +1,88 @@
+app-id: io.github.lenucksi.SieveEditor
+runtime: org.freedesktop.Platform
+runtime-version: '25.08'
+sdk: org.freedesktop.Sdk
+sdk-extensions:
+  - org.freedesktop.Sdk.Extension.openjdk21
+
+command: sieveeditor
+
+finish-args:
+  # X11 access for Swing GUI
+  - --socket=x11
+  # IPC namespace (required for X11)
+  - --share=ipc
+  # Network access for ManageSieve connections
+  - --share=network
+  # Legacy: Allow access to old profile location for migration
+  # - --filesystem=~/.sieveprofiles:ro
+  # Required to build
+  - --env=PATH=/app/jre/bin:/app/bin:/usr/bin
+  - --env=JAVA_HOME=/app/jre
+
+
+modules:
+  - name: openjdk
+    buildsystem: simple
+    build-commands:
+      - /usr/lib/sdk/openjdk21/install.sh
+
+  - name: sieveeditor
+    buildsystem: simple
+    build-options:
+      env:
+        PATH: /app/bin:/usr/bin:/usr/lib/sdk/openjdk21/bin
```

Comment:
use `prepend-path` or `append-path`

Reply:
Make a commit suggestion and I'm happy to integrate it.